docs/jenkins.md

```diff
@@ -54,6 +54,20 @@ https://github.com/theforeman/foreman-infra/tree/master/puppet/modules contains
 
 ### Configuring a new node
 
+For Enterprise Linux 9:
+
+* Ensure yum.puppet.com is configured: [puppet7-release](https://yum.puppet.com/puppet7-release-el-9.noarch.rpm)
+* `yum -y install puppet-agent`
+* `echo "server = puppet.theforeman.org" >> /etc/puppetlabs/puppet/puppet.conf`
+* ensure hostname is set node0X.jenkins.<provider\>.theforeman.org where <provider\> is osuosl or aws for example and that the record is in DNS
+* Make the `puppet` command available: `source /etc/profile.d/puppet-agent.sh`
+* `puppet ssl bootstrap`
+* Sign the certificate on the puppetmaster or via Foreman
+* `puppet agent --test`
+* Set the host group to "Builders" in Foreman
```

Comment:
is the "builders" group really needed these days? or should most things come from the `profile::jenkins::node` class?

Reply:
it's needed for the koji certs and user accounts

Reply:
> it's needed for the koji certs and user accounts

Is that all? That would imply we can put an issue to track dropping this step once we drop Koji.

Reply:
that was the only thing I've seen erroring out